hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -3297,6 +3297,7 @@ TermIndex installCheckpoint(String leaderId, Path checkpointLocation,
     if (canProceed) {
       // Stop RPC server before stop metadataManager
       omRpcServer.stop();
+      omRpcServer.join();
```

Comment:
Is this needed? 
`Server.java`
```
  public synchronized void stop() {
    LOG.info("Stopping server on " + this.port);
    this.running = false;
    if (this.handlers != null) {
      for(int i = 0; i < this.handlerCount; ++i) {
        if (this.handlers[i] != null) {
          this.handlers[i].interrupt();
        }
      }
    }

    this.listener.interrupt();
    this.listener.doStop();
    this.responder.interrupt();
    this.notifyAll();
    this.rpcMetrics.shutdown();
    this.rpcDetailedMetrics.shutdown();
  }
```
and 
```  
public synchronized void join() throws InterruptedException {
    while(this.running) {
      this.wait();
    }

  }
```

Reply:
Your are right, this is for other threads.